ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -68,12 +70,17 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
     id: 'givenName',
     header: 'Given name',
     accessorKey: 'profile.givenName'
+  }, {
+    id: 'contactEmail',
+    header: 'Contact email',
+    accessorKey: 'profile.contactEmail'
   }, {
     header: 'Consented',
-    accessorKey: 'enrollee.consented',
-    cell: info => info.getValue() ? <FontAwesomeIcon icon={faCheck}/> : ''
+    accessorFn: data => data.enrollee.consented.toString(),
```

Comment:
Converting this cell to text so we can filter on it. I attempted to handle this by adding a boolean dropdown filter but I couldn't get it working quickly and it made the table look horrible.

Reply:
Just caught up with Erin about this. I'm gonna do one more pass at trying to improve filtering on this column